manifests/init.pp

```diff
@@ -119,14 +119,12 @@
   # CCE-26785-6
   $at_boot = true,
   # CCE-26933-2
-  $to_syslog = defined('$::enable_logging')  ? { true => $::enable_logging,  default => hiera('enable_logging',true) },
+  $syslog = simplib::lookup('simp_options::syslog', { 'default_value' => false, 'value_type' => Boolean } ),
```

Comment:
Just make this strongly typed. You're doing it anyway with the `value_type` and it's both easier and you can ditch the `validate_bool` below if you do that. Not rejecting for this but it'll save a lot of headaches.